File: Python/1020.py
entrada = int(input())
anos = 0
meses = 0
while entrada >= 365:
    anos += 1
    entrada -= 365
while entrada >= 30:
    meses += 1
    entrada -= 30
print(anos, "ano(s)")
print(meses, "mes(es)")
print(entrada, "dia(s)")

File: Javascript/1020.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let dias = parseInt(lines.shift());
let meses = 0;
let anos = 0;
for (anos = 0; dias >= 365; anos++)
    dias -= 365;
for (meses = 0; dias >= 30; meses++)
    dias -= 30;
console.log(anos, "ano(s)\n" + meses, "mes(es)\n" + dias, "dia(s)");
